rllib/policy/tf_policy.py

```diff
@@ -155,14 +156,8 @@ def get_placeholder(self, name):
         if name in obs_inputs:
             return obs_inputs[name]
 
-        if not self.loss_initialized():
-            raise RuntimeError(
-                "You cannot call policy.get_placeholder() for non-obs inputs "
-                "before the loss has been initialized. To avoid this, use "
-                "policy.loss_initialized() to check whether this is the "
-                "case, or move the call to later (e.g., from stats_fn to "
-                "grad_stats_fn).")
-
+        assert self._loss_input_dict is not None, \
+            "Should have set this before get_placeholder can be called"
```

Comment:
This PR also fixes get_placeholder() to work even before the loss is initialized.